tests/functional/TestKubeSmokeTest/testHeketiRpc.sh

```diff
@@ -38,12 +38,14 @@ setup_all_pods() {
 	# Start Heketi
 	echo -e "\nStart Heketi container"
   sed 's\<ApiHost>\'"$KUBEAPI"'\g; s\<SecretName>\'"$KUBESEC"'\g' test-heketi-deployment.json | kubectl create -f - --validate=false || fail "Unable to start heketi container"
-	sleep 30
+	sleep 300
 
 	# This blocks until ready
 	kubectl expose deployment heketi --type=NodePort || fail "Unable to expose heketi service"
```

Comment:
AFAIK, this called blocked until the container was available.  Maybe line 40 never succeeded?